src/Place/ReadModel/Duplicate/CleanPlaceName.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\Duplicate;
+
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Model\ValueObject\Geography\Address;
+
+class CleanPlaceName
+{
+    private const MAX_LENGTH_TITLE = 150;
+    private const NONE = 'none';
+
+    public static function transform(Address $address, string $title): string
+    {
+        if (mb_strtolower($title) === self::NONE) {
```

Comment:
I guess this should match various casing?

Reply:
it does now? Or do you mean it should not?